docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -1,30 +1,106 @@
 
-* The system builds your application by using community dependencies.
+JBS automates this process as much as possible, making it much less time-consuming to create builds that are build from source in a controlled environment.
 
-* The JVM Build Service analyzes your application and determines the dependencies that the system needs to be rebuilt.
+=== Workflow Description
 
-* The JVM Build Service generates `ArtifactBuild` objects to represent each artifact in your application before determining to which repository the artifacts belong. The JVM Build Service tags from which systems the artifacts derive.
+When using the JVM Build Service the end user workflow is as follows:
 
-* The system analyzes the repository and uses the results to find a build strategy. For example, if the system cannot find the JDK version to use, the system uses all versions to determine the most effective version to use.  
+. The system builds your application by using community dependencies.
 
-* The system attempts to build all the dependencies and stores them in container images in an image registry, and `quay.io` is the default.
+. The JVM Build Service analyzes your application and determines the dependencies that the system needs to rebuild.
 
-* When the system has completed all its builds, you can rebuild your application.
+. The JVM Build Service generates `ArtifactBuild` objects to represent each artifact in your application before determining to which repository the artifacts belong. The JVM Build Service tags from which systems the artifacts derive.
 
-* If all the builds are successful, your application's SBOM displays that none of the dependencies came from third-party repositories. However, you may need to troubleshoot when some dependencies fail to build.
+. The system analyzes the repository and uses the results to find a build strategy. For example, if the system cannot find the JDK version to use, the system uses all versions to determine the most effective version to use.
 
-.*Procedure*
+. The system attempts to build all the dependencies and stores them in container images in an image registry, and `quay.io` is the default.
+
+. When the system has completed all its builds, you can rebuild your application.
+
+. If all the builds are successful, your application's SBOM displays that none of the dependencies came from third-party repositories. However, you may need to troubleshoot when some dependencies fail to build.
+
+== Setup
+
+=== Logging into the Server
+
+All interactions with the JVM Build Service require you to be logged into the OpenShift namespace used for your builds. This can be done by copying the login command from https://registration-service-toolchain-host-operator.apps.stone-prd-host1.wdlc.p1.openshiftapps.com/ .
+
+=== Setting up the CLI
```

Comment:
Minor tweaks based off Red Hat modularization standards:

=== Setting up the CLI 

Many of the commands can be setup using the JVM Build Service CLI. The CLI is currently distributed as a docker image. 

.Prerequisite 

- Log into your OpenShift namespace. 

.Procedure 

1. To use the docker image, create the following alias: < your provided alias >.  
2. To update the latest version, create an updated alias: <your provided updated alias >. 
3. To pull the latest version of the docker image, run the updated alias. 
4. To show the usage, run commands with a `--help` option. 
5. To  modify build recipes, like commands to fix failing builds, create a file like the following: < your provided command to create the file >.